ratbagd/ratbagd-device.c

```diff
@@ -223,6 +246,7 @@ ratbag_device_get_model(sd_bus *bus,
 const sd_bus_vtable ratbagd_device_vtable[] = {
 	SD_BUS_VTABLE_START(0),
 	SD_BUS_PROPERTY("Model", "s", ratbag_device_get_model, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+	SD_BUS_PROPERTY("DeviceType", "s", ratbagd_device_get_device_type, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
In general, string types should be used for something that's either free-form (like the model) or user-exposed (like the name). Where we have an enum, we should use unsigned ints instead - not having to write string parsing code is definitely a bonus :)

As a general rule: in C you avoid strings at all costs wherever you can, because they require too much work.

Reply:
ok but should we still send the strings "keyboard" or "mouse" over the DBus? or just send an int to represent the enum?

Reply:
Just the int, implement the names in Python.